### DISCUSSION TITLE
How do I stop seeing someone in the new feed?

### DISCUSSION THREAD

**Discussion by leoheck** [community/community#14495] [Other Feature Feedback, Questions, & Ideas]
The title says it all.

**Comment by adamdmharvey** [comment]
For me, if I highlight over an entry in the feed, the ellipses (`...`) shows up at the right and tells me why I see it. In most cases, it's because I follow the user, but sometimes it tells me it's because I'm following a repo. That should give enough insight to unfollow or unstar, thus removing from the feed?

e.g.,
<img width="772" alt="image" src="https://user-images.githubusercontent.com/33203301/162356225-1e92252d-8a4b-4eeb-b494-656ed47ea6cb.png">

**Reply by leoheck** [reply]
I will check. But I don't follow anyone. If it is related to the company O work. I don't want to see. Because it was not chosen by me. I wan to see things I like, and I follow. I don't want to see things that random robots think I want to see. These robots are push forcing things on users throats. I love GitHub. I don't love random, cold and brainless robots pushing streams of random content that I cannot even remove.

**Reply by adamdmharvey** [reply]
Your profile says you follow 4 people, so it may be those 4. :)

**Reply by leoheck** [reply]
![Screenshot_20220408-003410](https://user-images.githubusercontent.com/1277920/162357693-8d3616b6-cd23-40dd-974a-adc24646bf6a.png)

See the options I have here? I was interacting with him because I have no other choice. But I can only report the user. The user is not the problem. I just don't want to follow him and see his things.

**Reply by leoheck** [reply]
> Your profile says you follow 4 people, so it may be those 4. :)

No it is not.

**Reply by leoheck** [reply]
> Your profile says you follow 4 people, so it may be those 4. :)

But it is fine. Challenge was accepted. I will unfollow everyone. And I will still see these random people that I don't even want to see...

**Reply by leoheck** [reply]
GitHub is for code. It doesn't mean we have and want to socialize. If I want to socialize I can use Facebook, right? So, why I don't have any other choice to see whatever I want? This is my feed, I should not be able to see toxic things that I don't want there.

**Reply by leoheck** [reply]
> Your profile says you follow 4 people, so it may be those 4. :)

Alright my friend. I am not following anyone anymore.

**Reply by fredoist** [reply]
> But it is fine. Challenge was accepted. I will unfollow everyone. And I will still see these random people that I don't even want to see...

The new feed seems to show suggestions based on your user follows, starred repos and topics, and your own activity. So if someone adds a contribution to a repo you starred you'll see it, but you will also see suggestions of new repos or users based on your account activity.

Anyway it would be great to have a "don't want to see it" option like YouTube does for videos.

For now you can still switch to the old following feed which only shows activity on repos and users but doesn't add random suggestions.

**Comment by leoheck** [comment]
> Anyway it would be great to have a "don't want to see it" option like YouTube does for videos.

Exactly. 
Interacting with `code` is not the same as interacting with `people`.
This is not social media, and it should not be.

> For now you can still switch to the old following feed.

Sure, but this panel is far more useless... 
For instance, Github does not have a Body page, it just has 2 sidebars. 
For years, I barely look into the middle of the page since there is just useless content. And not just useless. Contains what I want to avoid seeing. If they put eye tracking on users, they will get interesting results about the void that this area is.

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)

**Reply by matthuska** [reply]
Hi @jdevstatic , that doesn't really address the original problem. The goal is to filter out specific individuals, repositories, etc. from the feed rather than removing whole classes of items (e.g. all "repositories"). I have a problem where I contributed to a large project, and the build bot for that project is on my feed now and basically spams it with a ton of activity. The only way I can remove those messages is to uncheck "Repositories", but that removes other repos from my feed and not just the problematic one.

As was stated earlier in the thread, we really need a per-item "not interested" option in the 3 dots menu next to each item (which currently only contains a reason the item is being displayed and "report").

**Reply by jfullstackdev** [reply]
yup, it's being worked on by the Staff

**Reply by flyingcakes85** [reply]
Hi, any ETA about this feature? I have the same requirements of being able to block certain users without completely removing all items.

**Reply by rongzhao-ambit** [reply]
Any ETA on this?

I am constantly receiving feeds from a user I don't want to see. So I want to block someone from my feed permantly! I don't want to block my entire category. I just want to block this user, only!
